docs/getting-started/zowe_faq.md

```diff
@@ -499,19 +499,19 @@ As a developer, you may contribute to Zowe Explorer in the following ways:
 
 </details>
 
-## Zowe IntelliJ plug-in FAQ
+## Zowe Explorer plug-in for IntelliJ IDEA FAQ
 
-### Why might I use Zowe IntelliJ plug-in versus a traditional ISPF interface to perform mainframe tasks?
+### Why might I use the plug-in versus a traditional ISPF interface to perform mainframe tasks?
```

Comment:
I think we should change the question to "Why **should** I use ... " or "Why **would** I use ... "